src/components/editor/EditAuthorsForm.jsx

```diff
@@ -60,19 +62,22 @@ export default class EditAuthorList extends React.Component {
     let newValue = {};
     newValue[slug] = false;
     let newAuthorsDeleted = Object.assign({}, this.state.authorsDeleted, newValue);
-    this.setState({
+    this.safeSetState({
       authorsDeleted: newAuthorsDeleted
     });
   }
 
+  handleStopPropagation(event) {
```

Comment:
No need to make a function that just does one line. Just do that one line immediately.